src/wwwroot/services/settings.js

```diff
@@ -161,5 +163,18 @@
       });
     }
 
+    function getCurrentPlanInfo() {
```

Comment:
mmm, I understand the scenario, but I am still not sure. For example, if we keep this cache, we should invalidate it when the user upgrades his account.

In my opinion, it could be a possible optimization in the future, but by the moment it looks as a premature optimization. In the same way, we could cache any request in the WebApp, and we are not doing that.

I think that the _List of Plans_ is a different scenario because the plans, does not change soooo much, and also it is a potentially expensive resource because it is a list of different plans.

Please, remove this cache, we could check how it works. But we could continue thinking in a way to reuse this logic and simplify cache invalidation, to use it when it will be necessary.